drodrpg/DRODLib/Character.cpp

```diff
@@ -2358,6 +2361,7 @@ void CCharacter::Process(
 			break;
 			case CCharacterCommand::CC_AddRoomToMap:
 			{
+				if (bRoomBeingDisplayedOnly) break;
```

Comment:
Thanks for making this change.
Recall lower down in the case's code block that `bProcessNextCommand = true;` needs to be executed before breaking in order to process the next commands. How about doing this instead:
`if (!bRoomBeingDisplayedOnly) { <do all of the code before that line> }`

Reply:
I didn't realize I had to set that value to true, let me amend everything accordingly.